frontend/pulse3d/src/pages/users-info.js

```diff
@@ -17,15 +18,35 @@ const customStyles = {
     style: {
       backgroundColor: "var(--dark-blue)",
       color: "white",
+      fontSize: "1.2rem",
     },
   },
   subHeader: {
     style: {
       backgroundColor: "var(--dark-blue)",
     },
   },
+  expanderButton: {
+    style: { flex: "0", margin: "0" },
+  },
+  rows: {
+    style: {
+      height: "60px",
+    },
+  },
+  cells: {
+    style: { padding: "0 0 0 1.3%" },
+  },
 };
 
+const modalObjs = {
+  deactivate: {
+    header: "Attention",
+    messages: [
+      "You are trying to deactivate some already inactive accounts. They will be ignored. Error Users :",
```

Comment:
I might change this  to 'You are attempting to deactivate accounts that are already inactive. ' or something